src/Uno.Foundation/UriExtensions.cs

```diff
@@ -26,6 +26,10 @@ public static class UriExtensions
 		}
 
 
+#if __ANDROID__
+		public static Uri TrimEndUriSlash(this Uri uri) => uri.AbsolutePath.EndsWith("/") ? new Uri(uri.ToString()[..^1]) : uri;
```

Comment:
Let's place this method internal for now. We'll see later if we need to make it public, as needed.

Reply:
If the method becomes internal will be broken unit tests and webAuthenticationBroker.

Reply:
Should work as internal. There's an `[assembly:InternalsVisibleTo]` attribute for that.

Reply:
Done!